apolloschurchapp/src/tabs/Tab.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useApolloClient, gql, useQuery } from '@apollo/client';
+
+import { BackgroundView, NavigationService } from '@apollosproject/ui-kit';
+import {
+  FeaturesFeedConnected,
+  FEATURE_FEED_ACTION_MAP,
+  RockAuthedWebBrowser,
+} from '@apollosproject/ui-connected';
+
+import { checkOnboardingStatusAndNavigate } from '@apollosproject/ui-onboarding';
+import { createNativeStackNavigator } from 'react-native-screens/native-stack';
+
+import { ONBOARDING_VERSION } from '../ui/Onboarding';
+
+function handleOnPress({ action, ...props }) {
+  if (FEATURE_FEED_ACTION_MAP[action]) {
+    FEATURE_FEED_ACTION_MAP[action]({ action, ...props });
+  }
+}
+
+export const createTab = ({ tabName, screenOptions, ...props }) => {
```

Comment:
Need to think really hard about what api and props this exposes. Right now I would say this feels inflexible. The Tab Component is really just a FeaturesFeed component, but if I didn't want a featuresfeed, I don't think I could change that?

Reply:
I think it's fine if it's inflexible, I just need to rename to `createFeatureFeedTab`